src/content/glossary/index.md

```diff
@@ -351,7 +354,7 @@ The ENS registry is a single central [contract](#smart-contract) that provides a
 
 ### execution client {#execution-client}
 
-Execution clients (f.k.a. "Eth1 clients"), such as Besu, Erigon, go-ethereum, Nethermind, are tasked with processing and broadcasting transactions, as well as with managing Ethereum's state. They run the computations for each transaction in the [Ethereum Virtual Machine](#evm) to ensure that the rules of the protocol are followed. Today, they also handle [proof-of-work](#pow) consensus. After the transition to [proof-of-stake](#pos), they will delegate this to consensus clients.
+Execution clients (f.k.a. "Eth1 clients"), such as Besu, Erigon, go-ethereum, Nethermind, are tasked with processing and broadcasting transactions, as well as with managing Ethereum's state. They run the computations for each transaction in the [Ethereum Virtual Machine](#evm) to ensure that the rules of the protocol are followed.
```

Comment:
I've never seen f.k.a used before (assuming it's formerly known as?). 

```suggestion
Execution clients (formerly known as "Eth1 clients"), such as Besu, Erigon, go-ethereum, Nethermind, are tasked with processing and broadcasting transactions and managing Ethereum's state. They run the computations for each transaction in the [Ethereum Virtual Machine](#evm) to ensure that the rules of the protocol are followed.
```

Reply:
Also not keen on naming specific clients here but not strongly opposed to keeping it if people feel otherwise.